setup.cfg

```
@@ -81,7 +81,7 @@ full =
     astropy>=4.0.1,<6.0.0
     f90nml>=1.1.2
     fastcache>=1.0.2
-    firefly-vis>=2.0.4,<3.0.0
+    firefly>=3.0.0
```

I would like to keep this as bullet-proof as possible, so that yt+firefly keeps working in CI even when API changes upstream. AFAIC I'm very happy with the current workflow where this kind of PR helps us migrate to newer API, but we do not depend.
```suggestion
    firefly>=3.2.0,<4.0.0
```